pkg/pg/collectors.go

```diff
@@ -85,32 +83,7 @@ func ActiveConnections(pgPool *pgxpool.Pool) func(ctx context.Context, state *ag
 			return err
 		}
 
-		metricEntry, err := utils.NewMetric(keywords.PGActiveConnections, result, utils.Int)
-		if err != nil {
-			return err
-		}
-		state.AddMetric(metricEntry)
-
-		return nil
-	}
-}
-
-const SleepQuery = `
-/*dbtune*/
-SELECT pg_sleep(1000000);
-`
-
-func ArtificiallyFailingQueries(pgPool *pgxpool.Pool) func(ctx context.Context, state *agent.MetricsState) error {
-	// Perform runtime reflection to make sure that the
-	// struct is embedding the default adapter
-	return func(ctx context.Context, state *agent.MetricsState) error {
-		var result int
-		err := pgPool.QueryRow(ctx, SleepQuery).Scan(&result)
-		if err != nil {
-			return err
-		}
-
-		metricEntry, err := utils.NewMetric("pg_sleep_result", result, utils.Int)
+		metricEntry, err := metrics.PGActiveConnections.AsFlatValue(result)
```

Comment:
Weird diff here, I just deleted the `SleepQuery` and `ArtificallyFailingQueries`, which it then merged the bottom of the function with.